src/example-types/termsDelta.js

```diff
@@ -1,74 +1,50 @@
-let _ = require('lodash/fp'),
-  aggUtils = require('../aggUtils')
-
-let example = {
-  config: {
-    field: 'Organization.NameState.untouched',
-    background: {
-      type: 'range',
-      field: 'PO.IssuedDate',
-      data: {
-        gte: 'now-1y-90d',
-        lte: 'now-90d',
-        format: 'dateOptionalTime'
-      }
-    },
-    foreground: {
-      type: 'range',
-      field: 'PO.IssuedDate',
-      data: {
-        gte: 'now-90d',
-        lte: 'now',
-        format: 'dateOptionalTime'
-      }
-    }
-  }
-}
```

Comment:
again, we should comment this out and not delete it until we can add it to the docs